docs/contributor_tips.md

```diff
@@ -0,0 +1,58 @@
+# Maintainer Tips
```

Comment:
Maintainer Tips or Contributor Tips? Title should match file name / PR description.

Reply:
Whoops fixed.